lib/transform/transformers.js

```diff
@@ -31,10 +31,7 @@ export function assets (asset) {
     asset.fields.file,
      (newFile, file, locale) => {
        newFile[locale] = omit(file, 'url', 'details')
```

Comment:
So `omit` is back again : ). Maybe we can move https://github.com/contentful/contentful-batch-libs/pull/94/files#diff-38a92f6d9573a0032767c762b9e9e235R35 here and then on https://github.com/contentful/contentful-batch-libs/pull/94/files#diff-38a92f6d9573a0032767c762b9e9e235R34 check for `if (file[locale].uploadFrom)`.